### ISSUE TITLE
Create binary packages of libnifalcon

### ISSUE THREAD

**Issue by qdot** [libnifalcon/libnifalcon#8]
Figure out what distributions should look like, and create binary distributables for libnifalcon so people will stop asking how to build it.

**Comment by agirault** [comment]
I recommend CPack if this is still considered.

**Comment by qdot** [comment]
Yup, wanted to go CPack with this, just never took the time to get it properly figured out back then. And now github has nicer release management too, so that should make things work well together. :D
